src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java

```diff
@@ -113,6 +113,16 @@ public long assignForUser(long userId, Levels levelName, LocalDateTime deadline)
         return test.getId();
     }
 
+    @Override
+    public void deassign(long id) {
```

Comment:
Shouldn't we update the test status after deassign?

Reply:
No. the status should not be `assigned` at the point